src/global_scheduler/global_scheduler.c

```diff
@@ -39,13 +39,10 @@ void assign_task_to_local_scheduler(global_scheduler_state *state,
             object_id_to_string(local_scheduler_id, id_string, ID_STRING_SIZE));
   task_set_state(task, TASK_STATUS_SCHEDULED);
   task_set_local_scheduler(task, local_scheduler_id);
-  retry_info retry = {
-      .num_retries = 0, .timeout = 100, .fail_callback = NULL,
-  };
   LOG_DEBUG("Issuing a task table update for task = %s",
             object_id_to_string(task_task_id(task), id_string, ID_STRING_SIZE));
   UNUSED(id_string);
-  task_table_update(state->db, copy_task(task), &retry, NULL, NULL);
+  task_table_update(state->db, copy_task(task), NULL, NULL, NULL);
```

Comment:
Hmm.. that would be fine as well, although I just intended `NULL` to mean "use the default.

We do check for `NULL` in `init_table_callback`.